### ISSUE TITLE
Crash Report Pixel 6 UI error 

### ISSUE THREAD

**Issue by AlexTu2** [TeamAmaze/AmazeFileManager#3518]
## Exception
* __App Name:__ Amaze File Manager
* __Package:__ com.amaze.filemanager
* __Version:__ 3.8.1
* __User Action:__ UI Error
* __Request:__ Application crash
* __OS:__ Linux Android 13 - 33
* __Device:__ oriole
* __Model:__ Pixel 6
* __Product:__ oriole
<details><summary><b>Crash log </b></summary><p>

```
io.reactivex.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.lang.StringIndexOutOfBoundsException: begin 6, end -1, length 24
	at io.reactivex.internal.observers.EmptyCompletableObserver.onError(EmptyCompletableObserver.java:50)
	at io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.onError(CompletableSubscribeOn.java:74)
	at io.reactivex.internal.operators.completable.CompletableAndThenCompletable$SourceObserver.onError(CompletableAndThenCompletable.java:62)
	at io.reactivex.internal.operators.completable.CompletableFromCallable.subscribeActual(CompletableFromCallable.java:40)
	at io.reactivex.Completable.subscribe(Completable.java:2309)
	at io.reactivex.internal.operators.completable.CompletableAndThenCompletable.subscribeActual(CompletableAndThenCompletable.java:35)
	at io.reactivex.Completable.subscribe(Completable.java:2309)
	at io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run(CompletableSubscribeOn.java:64)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.StringIndexOutOfBoundsException: begin 6, end -1, length 24
	at java.lang.String.checkBoundsBeginEnd(String.java:3942)
	at java.lang.String.substring(String.java:2467)
	at com.amaze.filemanager.database.UtilitiesDatabase$Companion.migratePasswordInUris(UtilitiesDatabase.kt:428)
	at com.amaze.filemanager.database.UtilitiesDatabase$Companion.access$migratePasswordInUris(UtilitiesDatabase.kt:109)
	at com.amaze.filemanager.database.UtilitiesDatabase$Companion$MIGRATION_5_6$1.migrate(UtilitiesDatabase.kt:474)
	at androidx.room.RoomOpenHelper.onUpgrade(RoomOpenHelper.java:99)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onUpgrade(FrameworkSQLiteOpenHelper.java:183)
	at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:416)
	at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:316)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableSupportDatabase(FrameworkSQLiteOpenHelper.java:151)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.java:112)
	at androidx.room.RoomDatabase.inTransaction(RoomDatabase.java:706)
	at androidx.room.RoomDatabase.assertNotSuspendingTransaction(RoomDatabase.java:483)
	at androidx.room.RoomDatabase.compileStatement(RoomDatabase.java:542)
	at androidx.room.SharedSQLiteStatement.createNewStatement(SharedSQLiteStatement.java:65)
	at androidx.room.SharedSQLiteStatement.getStmt(SharedSQLiteStatement.java:72)
	at androidx.room.SharedSQLiteStatement.acquire(SharedSQLiteStatement.java:87)
	at com.amaze.filemanager.database.daos.HistoryEntryDao_Impl$7.call(HistoryEntryDao_Impl.java:129)
	at com.amaze.filemanager.database.daos.HistoryEntryDao_Impl$7.call(HistoryEntryDao_Impl.java:126)
	at io.reactivex.internal.operators.completable.CompletableFromCallable.subscribeActual(CompletableFromCallable.java:36)
	... 12 more

```
</details>
<hr>

**Comment by VishnuSanal** [comment]
dupe #3488 & fixed on v3.8.2 with #3494

@AlexTu2 please update Amaze to the latest version

**Comment by AlexTu2** [comment]
> dupe #3488 & fixed on v3.8.2 with #3494
> 
> @AlexTu2 please update Amaze to the latest version

Great! Works as expected
